.github/actions/security-scan/action.yaml

```diff
@@ -1,6 +1,12 @@
 name: Security Scan
 description: "Run Go vulernability scan and Trivy repo scan"
 
+# TEMP FIX FOR: https://github.com/aquasecurity/trivy-action/issues/389
+inputs:
+  github-token:
+    description: 'GitHub token'
+    required: true
```

Comment:
Let's not make it required.

Reply:
Ah yeah my bad, empty env can use whatever they did before. Sure.

Reply:
done